integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -80,28 +80,31 @@ object MojoImplementation {
       file.toPath().toRealPath().toAbsolutePath()
     }
 
-    def resolveArtifact(artifact: Artifact, classifier: String = ""): Option[File] = try {
-      val suffix = if (classifier.nonEmpty) s":$classifier" else ""
-      log.info("Resolving artifact: " + artifact + suffix)
-      val request = new ArtifactRequest()
-      request.setArtifact(
-        new DefaultArtifact(
-          artifact.getGroupId(),
-          artifact.getArtifactId(),
-          classifier,
-          "jar",
-          artifact.getVersion()
+    def resolveArtifact(artifact: Artifact, classifierOpt: Option[String] = None): Option[File] =
+      try {
+        val classifier = classifierOpt.orElse(Option(artifact.getClassifier())).getOrElse("")
```

Comment:
This got reformatted, so best to use the `Hide whitespace changes` when reviewing.